packages/badge/README.md

```diff
@@ -7,18 +7,15 @@
 ```Javascript
 import Badge from '@leafygreen-ui/badge';
 
-  <Badge
-    variant='lightBlue'
-    className='my-badge'
-  >
-    New
-  </Badge>
+<Badge variant='Blue' className='my-badge'>
```

Comment:
Ditto here. Either Variant.Blue or 'blue'.

Reply:
🤦‍♂ Whoops – fixed in all instances.